scripts/dts/edtlib.py

```diff
@@ -141,6 +142,17 @@ def chosen_dev(self, name):
         # to_path() checks that the node exists
         return self._node2dev[chosen.props[name].to_path()]
 
+    def compat_devs(self, compat):
+        """
+        Returns list of Devices that match 'compat' with the compatible property
+        or None if there is not matching 'compat'
+        """
+
+        if compat not in self._compat2dev:
+            return []
+
+        return self._compat2dev[compat]
```

Comment:
Can probably skip this function and `_compat2dev` (see comment in `dt_compat_enabled()`).